Resync Navigation API WPT tests

RupinMittal opened the pull request:
#### ec2fbeb9c0349efd77bb3681a0423f08ebece85c
<pre>
Resync Navigation API WPT tests
<a href="https://bugs.webkit.org/show_bug.cgi?id=295664">https://bugs.webkit.org/show_bug.cgi?id=295664</a>
<a href="https://rdar.apple.com/155469733">rdar://155469733</a>

Reviewed by Youenn Fablet.

Upstream commit: web-platform-tests/wpt@f9fe9de

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/after-transition-new-navigation-before-commit-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/after-transition-push-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/after-transition-push.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/after-transition-reload-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/after-transition-reload.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/after-transition-replace-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/after-transition-replace.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/after-transition-traversal-commit-new-navigation-before-commit-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/after-transition-traverse-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/after-transition-traverse.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/after-transition-uncancelable-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/after-transition-window-stop-before-commit-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/commit-throws-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/commit-throws.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/multiple-intercept-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/w3c-import.log: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/defaultPrevented-window-stop-after-dispatch.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/event-constructor-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/event-constructor.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/intercept-handler-throws-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/intercept-history-pushState.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-anchor-cross-origin-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-anchor-cross-origin.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-anchor-download-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-anchor-download-userInitiated-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-anchor-download-userInitiated.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-anchor-download.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-anchor-fragment-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-anchor-fragment.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-anchor-same-origin-cross-document-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-anchor-same-origin-cross-document.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-anchor-userInitiated-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-anchor-userInitiated.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-anchor-with-target-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-anchor-with-target.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-destination-getState-back-forward.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-destination-getState-navigate-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-destination-getState-navigate.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-destination-getState-reload-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-destination-getState-reload.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-form-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-form-get-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-form-get.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-form-reload-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-form-reload.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-form-traverse-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-form-traverse.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-form-userInitiated-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-form-userInitiated.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-form-with-target-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-form-with-target.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-form.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-history-back-after-fragment-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-history-back-after-fragment.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-history-back-after-pushState-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-history-back-after-pushState.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-history-back-cross-document-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-history-back-cross-document.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-history-go-0-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-history-go-0.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-history-pushState-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-history-pushState.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-history-replaceState-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-history-replaceState.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-iframe-location-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-iframe-location.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-location-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-location.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-meta-refresh-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-meta-refresh.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-multiple-history-pushState-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-multiple-history-pushState.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-multiple-location-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-multiple-location.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-navigation-back-cross-document.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-navigation-back-same-document-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-navigation-back-same-document-in-iframe-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-navigation-back-same-document-in-iframe.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-navigation-back-same-document.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-navigation-navigate-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-navigation-navigate.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-svg-anchor-fragment-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-svg-anchor-fragment.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-to-srcdoc-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-to-srcdoc.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-window-open-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-window-open-self-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-window-open-self.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-window-open.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/replaceState-in-unload-then-remove-iframe-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/replaceState-in-unload-then-remove-iframe.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/signal-abort-window-stop.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-activation/activation-traverse-not-in-entries-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/disambigaute-forward-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/disambigaute-traverseTo-forward-multiple-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/resources/helpers.js:
(window.waitForAllLenient):
(window.assertBothRejectExactly.async t):
(window.assertBothRejectDOM.async t):
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/intercept-async.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-204-205-download-then-same-document-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-double-intercept.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-in-transition-finished.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-intercept-precommitHandler-redirect.html: Copied from LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-commit-after-transition-intercept.html.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-intercept-precommitHandler-redirect_currententrychange-expected.txt: Copied from LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-commit-after-transition-intercept_currententrychange-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-intercept-precommitHandler-redirect_no-currententrychange-expected.txt: Copied from LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-commit-after-transition-intercept_currententrychange-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-intercept-precommitHandler-reject.tentative.html: Copied from LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-same-document-intercept-reject.html.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-intercept-precommitHandler-reject.tentative_currententrychange-expected.txt: Copied from LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-commit-after-transition-intercept_currententrychange-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-intercept-precommitHandler-reject.tentative_no-currententrychange-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-commit-after-transition-intercept_no-currententrychange-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-intercept-precommitHandler.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-commit-after-transition-intercept.html.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-intercept-precommitHandler_currententrychange-expected.txt: Copied from LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-commit-after-transition-intercept_currententrychange-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-intercept-precommitHandler_no-currententrychange-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-commit-after-transition-intercept_currententrychange-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-intercept-stop.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-intercept.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-same-document-intercept-reentrant.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-same-document-intercept-reject.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/reload-intercept-reject.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/reload-intercept.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/reload-no-popstate-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/reload-no-popstate.html: Copied from LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/reload-intercept.html.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/resources/helpers.mjs:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/multiple-intercept-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/multiple-intercept.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/multiple-intercept.html.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-new-navigation-before-commit-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-new-navigation-before-commit.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/after-transition-new-navigation-before-commit.html.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-push-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-push.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-redirect-options-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-redirect-options.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-redirect-push-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-redirect-push.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-redirect-replace-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-redirect-replace.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-redirect-throws-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-redirect-throws.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-reload-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-reload.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-replace-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-replace.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-traversal-commit-new-navigation-before-commit-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-traversal-commit-new-navigation-before-commit.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/after-transition-traversal-commit-new-navigation-before-commit.html.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-traversal-window-stop-before-commit-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-traversal-window-stop-before-commit.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-traverse-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-traverse.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-uncancelable-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-uncancelable.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/after-transition-uncancelable.html.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-window-stop-before-commit-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-window-stop-before-commit.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/after-transition-window-stop-before-commit.html.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/resources/precommitHandler-helpers.js: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/resources/after-transition-commit-helpers.js.
(destinationIndex.0.let.navigate_helpers.rejectBeforeCommit.async e):
(destinationIndex.0.let.navigate_helpers.rejectAfterCommit.async e):
(destinationIndex.0.let.navigate_helpers.success.async e):
(destinationIndex.0.async once):
(window.testDeferredCommit.async t):
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/resources/w3c-import.log: Copied from LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/resources/w3c-import.log.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/manual-scroll-before-after-transition-commit-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/manual-scroll-in-precommit-handler-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/manual-scroll-in-precommit-handler.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/manual-scroll-before-after-transition-commit.html.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/cross-document-away-and-back.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/cross-document-away-and-back_method=navigate-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/cross-document-away-and-back-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/cross-document-away-and-back_method=updateCurrentEntry-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/updateCurrentEntry-method/cross-document-away-and-back-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/cross-document-location-api.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/cross-document-location-api_method=navigate-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/cross-document-location-api-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/cross-document-location-api_method=updateCurrentEntry-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/updateCurrentEntry-method/cross-document-location-api-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/history-pushState.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/history-pushState_method=navigate-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/history-pushState-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/history-pushState_method=updateCurrentEntry-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/updateCurrentEntry-method/history-pushState-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/history-replaceState.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/history-replaceState_method=navigate-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/history-replaceState-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/history-replaceState_method=updateCurrentEntry-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/updateCurrentEntry-method/history-replaceState-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/location-reload.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/location-reload_method=navigate-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/location-reload-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/location-reload_method=updateCurrentEntry-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/updateCurrentEntry-method/location-reload-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/resources/helpers.js: Added.
(window.updateStateBasedOnTestVariant):
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/resources/w3c-import.log: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/resources/w3c-import.log.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/updateCurrentEntry-method/cross-document-away-and-back.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/updateCurrentEntry-method/cross-document-location-api.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/updateCurrentEntry-method/history-pushState.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/updateCurrentEntry-method/history-replaceState.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/updateCurrentEntry-method/location-reload.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/updateCurrentEntry-method/w3c-import.log:
* LayoutTests/platform/ios/TestExpectations:
* LayoutTests/platform/mac-wk2/TestExpectations:
* LayoutTests/platform/mac/TestExpectations:

Canonical link: <a href="https://commits.webkit.org/297418@main">https://commits.webkit.org/297418@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/aeca9f28e196f46cec15cace46e80b382af643da

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/111703 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/31369 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/21844 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/117736 "Built successfully") | [  ~~🛠 win~~](https://ews-build.webkit.org/#/builders/59/builds/61970 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/32057 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/39954 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/84876 "Passed tests") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/59/builds/61970 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/114650 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/25594 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/100535 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/65312 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/24934 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/18666 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/61573 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/94985 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/18738 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/120993 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/38753 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/28808 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/93767 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/39130 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/96789 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/93589 "Passed tests") | 
| | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/38734 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/16531 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/34787 "Built successfully") | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/17998 "Built successfully and passed tests") | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/38643 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/38303 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/41634 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/40007 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/4d5630b39b52eecdb92abf589d98887ed6b606fe. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/47787

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/aabf65f9bb5e190c4de2d50e40f0e373ec4aa3a4. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/47787

RupinMittal commented:
> > The mac-wk2-stress bot is timing out. The mac-wk2 is green so one run of the tests has succeeded, which I imagine would mean the expectations are set correctly. My guess is that the 100 iterations being run by the stress bot is just taking too long.
> > I've asked about this on Slack to see if it's safe to land.
> 
> As discussed on Slack, we believe this is ok to land. I'm adding it to the merge queue.

As discussed on Slack, we need to skip some more tests. I've done that, hopefully the bots will all be green now.

RupinMittal committed:
Resync Navigation API WPT tests
https://bugs.webkit.org/show_bug.cgi?id=295664
rdar://155469733

Reviewed by Youenn Fablet.

Upstream commit: web-platform-tests/wpt@f9fe9de

* LayoutTests/TestExpectations:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/after-transition-new-navigation-before-commit-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/after-transition-push-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/after-transition-push.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/after-transition-reload-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/after-transition-reload.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/after-transition-replace-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/after-transition-replace.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/after-transition-traversal-commit-new-navigation-before-commit-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/after-transition-traverse-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/after-transition-traverse.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/after-transition-uncancelable-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/after-transition-window-stop-before-commit-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/commit-throws-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/commit-throws.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/multiple-intercept-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/w3c-import.log: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/defaultPrevented-window-stop-after-dispatch.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/event-constructor-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/event-constructor.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/intercept-handler-throws-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/intercept-history-pushState.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-anchor-cross-origin-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-anchor-cross-origin.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-anchor-download-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-anchor-download-userInitiated-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-anchor-download-userInitiated.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-anchor-download.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-anchor-fragment-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-anchor-fragment.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-anchor-same-origin-cross-document-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-anchor-same-origin-cross-document.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-anchor-userInitiated-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-anchor-userInitiated.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-anchor-with-target-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-anchor-with-target.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-destination-getState-back-forward.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-destination-getState-navigate-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-destination-getState-navigate.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-destination-getState-reload-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-destination-getState-reload.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-form-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-form-get-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-form-get.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-form-reload-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-form-reload.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-form-traverse-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-form-traverse.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-form-userInitiated-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-form-userInitiated.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-form-with-target-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-form-with-target.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-form.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-history-back-after-fragment-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-history-back-after-fragment.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-history-back-after-pushState-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-history-back-after-pushState.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-history-back-cross-document-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-history-back-cross-document.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-history-go-0-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-history-go-0.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-history-pushState-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-history-pushState.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-history-replaceState-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-history-replaceState.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-iframe-location-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-iframe-location.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-location-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-location.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-meta-refresh-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-meta-refresh.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-multiple-history-pushState-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-multiple-history-pushState.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-multiple-location-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-multiple-location.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-navigation-back-cross-document.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-navigation-back-same-document-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-navigation-back-same-document-in-iframe-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-navigation-back-same-document-in-iframe.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-navigation-back-same-document.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-navigation-navigate-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-navigation-navigate.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-svg-anchor-fragment-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-svg-anchor-fragment.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-to-srcdoc-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-to-srcdoc.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-window-open-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-window-open-self-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-window-open-self.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/navigate-window-open.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/replaceState-in-unload-then-remove-iframe-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/replaceState-in-unload-then-remove-iframe.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/signal-abort-window-stop.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-activation/activation-traverse-not-in-entries-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/disambigaute-forward-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/disambigaute-traverseTo-forward-multiple-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/return-value/resources/helpers.js:
(window.waitForAllLenient):
(window.assertBothRejectExactly.async t):
(window.assertBothRejectDOM.async t):
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigation-methods/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/intercept-async.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-204-205-download-then-same-document-expected.txt:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-double-intercept.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-in-transition-finished.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-intercept-precommitHandler-redirect.html: Copied from LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-commit-after-transition-intercept.html.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-intercept-precommitHandler-redirect_currententrychange-expected.txt: Copied from LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-commit-after-transition-intercept_currententrychange-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-intercept-precommitHandler-redirect_no-currententrychange-expected.txt: Copied from LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-commit-after-transition-intercept_currententrychange-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-intercept-precommitHandler-reject.tentative.html: Copied from LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-same-document-intercept-reject.html.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-intercept-precommitHandler-reject.tentative_currententrychange-expected.txt: Copied from LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-commit-after-transition-intercept_currententrychange-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-intercept-precommitHandler-reject.tentative_no-currententrychange-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-commit-after-transition-intercept_no-currententrychange-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-intercept-precommitHandler.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-commit-after-transition-intercept.html.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-intercept-precommitHandler_currententrychange-expected.txt: Copied from LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-commit-after-transition-intercept_currententrychange-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-intercept-precommitHandler_no-currententrychange-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-commit-after-transition-intercept_currententrychange-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-intercept-stop.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-intercept.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-same-document-intercept-reentrant.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/navigate-same-document-intercept-reject.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/reload-intercept-reject.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/reload-intercept.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/reload-no-popstate-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/reload-no-popstate.html: Copied from LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/reload-intercept.html.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/resources/helpers.mjs:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/ordering-and-transition/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/multiple-intercept-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/multiple-intercept.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/multiple-intercept.html.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-new-navigation-before-commit-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-new-navigation-before-commit.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/after-transition-new-navigation-before-commit.html.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-push-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-push.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-redirect-options-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-redirect-options.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-redirect-push-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-redirect-push.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-redirect-replace-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-redirect-replace.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-redirect-throws-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-redirect-throws.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-reload-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-reload.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-replace-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-replace.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-traversal-commit-new-navigation-before-commit-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-traversal-commit-new-navigation-before-commit.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/after-transition-traversal-commit-new-navigation-before-commit.html.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-traversal-window-stop-before-commit-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-traversal-window-stop-before-commit.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-traverse-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-traverse.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-uncancelable-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-uncancelable.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/after-transition-uncancelable.html.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-window-stop-before-commit-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/precommitHandler-window-stop-before-commit.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/after-transition-window-stop-before-commit.html.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/resources/precommitHandler-helpers.js: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/resources/after-transition-commit-helpers.js.
(destinationIndex.0.let.navigate_helpers.rejectBeforeCommit.async e):
(destinationIndex.0.let.navigate_helpers.rejectAfterCommit.async e):
(destinationIndex.0.let.navigate_helpers.success.async e):
(destinationIndex.0.async once):
(window.testDeferredCommit.async t):
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/resources/w3c-import.log: Copied from LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/resources/w3c-import.log.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/precommit-handler/w3c-import.log: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/manual-scroll-before-after-transition-commit-expected.txt: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/manual-scroll-in-precommit-handler-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/manual-scroll-in-precommit-handler.html: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/manual-scroll-before-after-transition-commit.html.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/scroll-behavior/w3c-import.log:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/cross-document-away-and-back.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/cross-document-away-and-back_method=navigate-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/cross-document-away-and-back-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/cross-document-away-and-back_method=updateCurrentEntry-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/updateCurrentEntry-method/cross-document-away-and-back-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/cross-document-location-api.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/cross-document-location-api_method=navigate-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/cross-document-location-api-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/cross-document-location-api_method=updateCurrentEntry-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/updateCurrentEntry-method/cross-document-location-api-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/history-pushState.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/history-pushState_method=navigate-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/history-pushState-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/history-pushState_method=updateCurrentEntry-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/updateCurrentEntry-method/history-pushState-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/history-replaceState.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/history-replaceState_method=navigate-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/history-replaceState-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/history-replaceState_method=updateCurrentEntry-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/updateCurrentEntry-method/history-replaceState-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/location-reload.html:
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/location-reload_method=navigate-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/location-reload-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/location-reload_method=updateCurrentEntry-expected.txt: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/updateCurrentEntry-method/location-reload-expected.txt.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/resources/helpers.js: Added.
(window.updateStateBasedOnTestVariant):
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/state/resources/w3c-import.log: Renamed from LayoutTests/imported/w3c/web-platform-tests/navigation-api/commit-behavior/resources/w3c-import.log.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/updateCurrentEntry-method/cross-document-away-and-back.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/updateCurrentEntry-method/cross-document-location-api.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/updateCurrentEntry-method/history-pushState.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/updateCurrentEntry-method/history-replaceState.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/updateCurrentEntry-method/location-reload.html: Removed.
* LayoutTests/imported/w3c/web-platform-tests/navigation-api/updateCurrentEntry-method/w3c-import.log:
* LayoutTests/platform/ios/TestExpectations:
* LayoutTests/platform/mac-wk2/TestExpectations:
* LayoutTests/platform/mac/TestExpectations:

Canonical link: https://commits.webkit.org/297418@main
